editor/run/game_view_plugin.cpp

```diff
@@ -734,6 +743,84 @@ void GameView::_camera_override_button_toggled(bool p_pressed) {
 	debugger->set_camera_override(p_pressed);
 }
 
+void GameView::_debug_draw_override_menu_id_pressed(int p_id) {
```

Comment:
This handler can also be a static method in Node3DEditorViewport. The menus and callback can be arguments.